migrations/20200419205756_create_location.js

```diff
@@ -1,8 +1,10 @@
-/* migrations/[...]_create-location.js */
+/* migrations/[...]_create_location.js */
 
 exports.up = function(knex) {
   return knex.schema.createTable('location', (table) => {
-    table.string('weather_zipcode', 5).unsigned().notNullable().primary().references('zipcode').inTable('weather'); /* PK / FK (not sure if this is correct)*/
+    table.increments('id'); // PK 
+    table.integer('user_id').references('id').inTable('users'); // FK 
+    table.string('weather_zipcode', 5);
```

Comment:
I'm going to ask James tomorrow about the best way to store weather/location/lat,long./zip_code

Reply:
Sounds good!